helm/cluster-azure/values.schema.json

```diff
@@ -118,18 +156,22 @@
                 "defaults": {
                     "properties": {
                         "evictionMinimumReclaim": {
+                            "default": "imagefs.available=5%,memory.available=100Mi,nodefs.available=5%",
                             "title": "Default settings for eviction minimum reclaim",
                             "type": "string"
                         },
                         "hardEvictionThresholds": {
+                            "default": "memory.available\u003c200Mi,nodefs.available\u003c10%,nodefs.inodesFree\u003c3%,imagefs.available\u003c10%,pid.available\u003c20%",
```

Comment:
So `<` is becoming `\u003c` ? Is that necessary, or can it be avoided? I think it might make the schema slightly harder to maintain.

Reply:
Interesting, I didn't realize that. Seems like `schemalint normalize` is the reason that happens.

I will create an issue for that.

Reply:
Now that https://github.com/giantswarm/schemalint/pull/41 is merged, can we get rid of the escaping here in the schema please?